### ISSUE TITLE
Mode Symbols do not display on the mode line when in GUI mode

### ISSUE THREAD

**Issue by techno156** [syl20bnr/spacemacs#13781]
#### Description :octocat:
Mode Line symbols do not display correctly in the mode line, although they are shown correctly in the pop-up when clicked. This issue only occurs when in GUI mode.

#### Reproduction guide :beetle:
- Start Emacs
- Open File in buffer

*Observed behaviour:* :eyes: :broken_heart:
Unicode mode Symbols do not display correctly

*Expected behaviour:* :heart: :smile:
Unicode mode symbols are displayed correctly

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 39de9ff0a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(latex spotify pdf selectric emacs-lisp git helm multiple-cursors spell-checking syntax-checking treemacs)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS GLIB NOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM THREADS LIBSYSTEMD LCMS2

#### Screenshot :computer:
![Screenshot from 2020-07-18 02-14-59](https://user-images.githubusercontent.com/2368911/87809880-749ca680-c89f-11ea-9051-2ed357b660e5.png)

**Comment by techno156** [comment]
Running on Ubuntu 20.04 LTS.

**Comment by brandonwillard** [comment]
I'm also on 20.04 and recently had a similar issue with the icon for `helm-ff-cache-mode`.  Installing `fonts-symbola`&mdash;like [the ArchWiki suggests](https://wiki.archlinux.org/index.php/Spacemacs#Incorrect_minor_mode_icon_fonts)&mdash;seems to have fixed it.

**Comment by techno156** [comment]
> I'm also on 20.04 and recently had a similar issue with the icon for `helm-ff-cache-mode`. Installing `fonts-symbola`—like [the ArchWiki suggests](https://wiki.archlinux.org/index.php/Spacemacs#Incorrect_minor_mode_icon_fonts)—seems to have fixed it.

Unfortunately, that doesn't seem to work for me. `helm-ff-cache-mode` shows up as the white cube in the screenshot in the original issue, although I was under the impression that that was intentional, since the icon does not change between CLI and GUI modes, and neither does it change if the symbol is opened in the popup menu, unlike the mode symbols.

**Comment by brandonwillard** [comment]
Yeah, it didn't work right away for me, but after it did start working it shows a color changing light bulb.  Perhaps that's not what it's supposed to show, and I just lucked-out!
I've been checking my command history to see what else I might've done, but haven't found anything yet.

**Comment by duianto** [comment]
I don't know what's causing the mode line indicators to only appear as circles for you.

For me they appear correctly (a circle with a letter in it).

Tested with:
- PopOS 20.04 with the fonts:
  - Source Code Pro
  - Fira Mono (PopOS 20.04 default Emacs font)
- Windows 1903 with the fonts:
  - Source Code Pro
  - Courier New (Windows 10 default Emacs font)

The default font was observed by starting: `emacs -q`
and opening the menu: `Options` > `Set Default Font...`

#### helm-ff-cache-mode

https://github.com/emacs-helm/helm/commits/master
recently made some changes to `helm-ff-cache-mode`

That either added or made it's mode-line indicator appear.

With some fonts (including the Spacemacs suggested: Source Code Pro),
the indicator appeared as a Square with the characters:
```
01F
4A1
```
![emacs_2020-07-14_09-51-24](https://user-images.githubusercontent.com/13420573/87872122-52497c80-c9b6-11ea-80a2-a41a70f634f1.png)

Searching for the characters found that it's the unicode symbol: 💡
electric light bulb (U+1F4A1)
source: https://graphemica.com/%F0%9F%92%A1

An issue was opened on the helm repository about it:
Uncommon symbol on default helm-ff-cache-mode-lighter value https://github.com/emacs-helm/helm/issues/2341

The mode-line indicator was then changed to:
Add two new user var to customize helm-ff-cache-mode lighter ([#2341](https://github.com/emacs-helm/helm/issues/2341)) · emacs-helm/helm@319dfb6
- a square when helm-ff-cache is idle

![emacs_2020-07-17_13-28-13](https://user-images.githubusercontent.com/13420573/87872254-2975b700-c9b7-11ea-9049-64730f243491.png)

- and to a down pointing arrow when it's updating

![emacs_2020-07-17_13-28-17](https://user-images.githubusercontent.com/13420573/87872257-32ff1f00-c9b7-11ea-81e1-6152c54076c7.png)

The mode line indicator is now hidden by default in Spacemacs,
[helm] Hide helm-ff-cache-mode mode-line indicator #13774
